test/unit/event-handler-receive-test.js

```diff
@@ -1,10 +1,9 @@
-const test = require("tap").test;
-
-const receive = require("../../src/event-handler/receive");
+import { test } from "tap";
+import { receiverHandle as receive } from "../../pkg/dist-src/event-handler/receive";
```

Comment:
oh is the reason we need the `babelrc` because we want to test internal functionality like this? 

And once we rewrite the tests to TypeScript, we can get rid of babelrc again? I wonder if in this case it'd be simpler to rewrite the tests to TypeScript at the same time ...

Reply:
> oh is the reason we need the `babelrc` because we want to test internal functionality like this?
> 
> And once we rewrite the tests to TypeScript, we can get rid of babelrc again? I wonder if in this case it'd be simpler to rewrite the tests to TypeScript at the same time ...

As we commented [here](https://github.com/octokit/webhooks.js/pull/113#issuecomment-612578679) it would imply migrating from tap to jest + TypeScript. Don't you think is better to do it in 2 steps to not introduce more potential friction on these breaking changes? What do you think? @gr2m

Reply:
Got it, thanks for reminding me. Okay, let's use babelrc then until we migrate to Jest + TypeScript 👍 

I'll test your changes locally now